package/MDAnalysis/analysis/encore/clustering/ClusterCollection.py

```diff
@@ -215,7 +219,7 @@ def get_ids(self):
         ids : list of int
         list of cluster ids
         """
-        return [v.idn for v in self.clusters]
```

Comment:
This change needs a test, typos like this would have been caught

Reply:
Thanks - I've added a test for that and a couple of other explicit tests for similar methods of the class